fluent-kafka-streams-tests/src/test/java/com/bakdata/fluent_kafka_streams_tests/test_applications/WordCount.java

```diff
@@ -49,7 +50,7 @@ public Topology getTopology() {
         final KTable<String, Long> wordCounts = textLines
                 .flatMapValues(value -> Arrays.asList(pattern.split(value.toLowerCase())))
                 .groupBy((key, word) -> word)
-                .count();
+                .count(Materialized.as("count"));
```

Comment:
Do we still have a test that covers the previous naming starting with `KStream`?